Extensions/Physics2Behavior/physics2runtimebehavior.ts

```diff
@@ -673,7 +685,8 @@ namespace gdjs {
       return this._body;
     }
 
-    createBody() {
+    createBody(): boolean {
+      if (!this.activated() || this.destroyedDuringFrameLogic) return false;
```

Comment:
Could you explain to me what is the purpose of `this.destroyedDuringFrameLogic` please? :)
Do you have a case in mind when createBody would be called while `this.destroyedDuringFrameLogic` is `true`?

Reply:
When going through events, you can have, in order:
- A. If XXX, apply impulse to Object
- B. if YYY, destroy Object
- C. if ZZZ, set linear damping of Object

If all three conditions are true, when going through B, you destroy the object, and so you do for the box2d body.
When going through C, if you don't know that you destroyed the body, you will recreate it, leaving a ghost body (behavior without a runtime object linked to it).

And keep in mind that instead of having a C event, this can happen during post events.

Reply:
I think that no action can be called on a destroyed instance because destroyed instances can't be picked.

Reply:
Oh I didn't think of that.